### PR TITLE
[6932] contrast checking

### PR DESCRIPTION
**Pull request by khamui**
@philli-m tried to implement in Sass, but no chance to get a variable dynamically into mixins or functions.

so I went with a little JS script. Think it is good enough and we can get rid of the dependency.

### CODE REVIEW

File: apps/academy/templates/academy/academy_landing_page.html [hunk 1]
```diff
@@ -7,13 +7,17 @@
...
+    data-contrast-checking="{{ page.rgb_list }}"
+    style="background-image: linear-gradient(148deg, {{ page.color1 }}, {{ page.color2 }});"
+>
     <div class="container">
         <div class="row justify-content-center">
             <div class="col-lg-10">
                 <h1>{{ page.translated_intro_text }}</h1>
                 <div class="mt-md-5">
-                    <a style="color: {{ page.textcolor }};" href="{{ page.intro_link.url }}">{{ page.translated_intro_link_text }}</a>
+                    <a style="href="{{ page.intro_link.url }}">{{ page.translated_intro_link_text }}</a>
```
Comment (fuzzylogic2000) [inline apps/academy/templates/academy/academy_landing_page.html:20]: Is there soething missing here? `style="href="{{ pag....` does not look right.

**Review by fuzzylogic2000 — COMMENTED** [review]
It works! :tada: 
But I don't really get all of it, so @philli-m should also check. :)
And maybe you can clean up the commits a bit, to me the changes do make more sense if looked at alltogether than in single commits here.

**Comment by fuzzylogic2000** [comment]
On the landing page, there is the small nöppel on the upper left corner now (and it should only be there at other times). I don't have that on the current main, and here I only get it after installing, so maybe sth. with the removal of that package?
image

File: apps/academy/templates/academy/academy_landing_page.html (hunk 1 above)
Comment (khamui) [reply apps/academy/templates/academy/academy_landing_page.html:20]: uff sketchy. thanks for catching

**Review by philli-m — COMMENTED** [review]
sorry forgot to post it

**Review by philli-m — APPROVED** [review]
Nice!

**Pull request merged**